flexmeasures/data/queries/annotations.py

```diff
@@ -8,26 +8,32 @@
 from flexmeasures.data.models.annotations import (
     Annotation,
     GenericAssetAnnotationRelationship,
+    SensorAnnotationRelationship,
 )
 from flexmeasures.data.models.data_sources import DataSource
 
 
 def query_asset_annotations(
     asset_id: int,
+    sensor_id: int,
```

Comment:
We don't need to pass both an asset ID and a sensor ID, so I suggest we refactor to use one variable named `asset_or_sensor_id`.